### DISCUSSION TITLE
Future features

### DISCUSSION THREAD

**Discussion by gerardbosch** [zdharma-continuum/zinit#181] [Q&A] [answered]
Hi there! Small question for the maintainers:

Is the goal of this project (as a fork from the original RIP one) to be maintained and actively evolve with new features, improvements, etc? Or only bug-fixing will be addressed?

Thank you :)

**Comment by pschmitt** [comment, marked as answer]
This has been answered already but yeah we are planning new features, releases etc. We have also added a few new things since the inception. Just look at the commit log.

**Reply by gerardbosch** [reply]
Cool, thanks for clarifying :)
